chat-plugins/hangman.js

```diff
@@ -225,7 +225,7 @@ exports.commands = {
 
 		guess: function (target, room, user) {
 			if (!target) return this.parse('/help guess');
-			if (!room.game || room.game.gameType !== 'hangman') return this.errorReply("There is no game of hangman running in this room.");
+			if (!room.game || room.game.title !== 'Hangman') return this.errorReply("There is no game of hangman running in this room.");
```

Comment:
[12:13:26] Zarel: I don't think it's possible to make a better way
[12:13:32] Zarel: check to see if room.game already exists first
[12:13:39] Zarel: room.game.title should exist

gdi zarel